sonar-core/src/main/java/org/sonar/core/issue/DefaultIssueBuilder.java

```diff
@@ -77,6 +77,24 @@ public DefaultIssueBuilder message(@Nullable String s) {
   }
 
   @Override
+  public NewIssueLocation newLocation() {
+    // TODO
+    return null;
+  }
+
+  @Override
+  public IssueBuilder addExecutionFlow(NewIssueLocation... flow) {
+    // TODO
+    return null;
+  }
+
+  @Override
+  public IssueBuilder addLocation(NewIssueLocation location) {
+    // TODO
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)